src/plugins/sync-title/client.js

```diff
@@ -33,15 +33,22 @@ function installClientMutation(sync) {
 
 }
 
-module.exports = function clientSyncTitle(client) {
+module.exports = function (syncPage) {
+function clientSyncTitle(client) {
 
 	function sync(value) {
-		client.sendEvent('title', value);
+		if (syncPage) {
```

Comment:
If we don't mind duplicate code, we could also just make a separate  'iframe-title' client and host.